src/s2/s2region_coverer.h

```diff
@@ -28,6 +28,10 @@
 #include "s2/s2cell_id.h"
 #include "s2/s2cell_union.h"
 
+#ifdef _WIN32
+#pragma warning(disable : 4200)
```

Comment:
Instead of disabling the warning, can we change the code, maybe deleting the copy-ctor/copy-assign it's complaining about, or using std::array with 0 size instead (not sure if that's legal, just throwing it out there as an idea)?